repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st

```diff
@@ -0,0 +1,4 @@
+actions
+readerClassForFileNamed: fileName
+	^ "MCReader readerClassForFileNamed: fileName"
+		MCFileTreeReader
```

Comment:
Looks like this method needs to return MCReader otherewise I'm afraid that the other implementations of MCFileTreeRepository will be broken ... presumably MCFileTreeReader will be reading the Cypress format exclusively?